src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -154,9 +154,9 @@ const ProgressiveProfiling = (props) => {
       'edx.bi.welcome.page.submit.clicked',
       {
         isGenderSelected: !!values.gender,
-        isYearOfBirthSelected: !!values.year_of_birth,
-        isLevelOfEducationSelected: !!values.level_of_education,
-        isWorkExperienceSelected: !!values.work_experience,
+        isYearOfBirthSelected: !!values.yearOfBirth,
+        isLevelOfEducationSelected: !!values.levelOfEducation,
+        isWorkExperienceSelected: !!values.workExperience,
```

Comment:
please check for these occurances in other places too.

For example the <RedirectLogistration> component rendering in this file.